### ISSUE TITLE
Exported linux AppImage crashes

### ISSUE THREAD

**Issue by cjdg** [4ian/GDevelop#1234]
## Describe the bug
After building both web and local AppImage of plataformer example,  both versions crashes and  launch an core generated.

## To Reproduce
1.- Export plataformer game both web building or local.
2.- Build local version
3.- Run

* Please include a link to a game if possible!
* If applicable, add screenshots to help explain your problem.

## Other details
* Ubuntu Studio 19.04 64bit

**Comment by 4ian** [comment]
Can you include any output if you launch it from the terminal? Can you paste here any generated/dump file during the crash report?

**Comment by probonopd** [comment]
Hello, someone from the AppImage team here. Can you please provide an AppImage for download so that we can analyze this?

**Comment by 4ian** [comment]
@probonopd Attached is one I just made from the "Platformer" template:
https://www.dropbox.com/s/a89xr9pz80j9w9c/game-linux.AppImage?dl=0

**Comment by probonopd** [comment]
Thanks. From a first glance this seems to run rather well for me:

![](https://user-images.githubusercontent.com/2480569/76710812-64fb2e80-6702-11ea-9049-fdb76157da7f.png)

What is the issue?

**Comment by probonopd** [comment]
Minor nitpick, it would be nice if it would show the game icon.

Currently it is in `<AppDir>/usr/share/icons/hicolor/0x0/apps/platformer.png` which is wrong. It should be in, e.g.,  `<AppDir>/usr/share/icons/hicolor/512x512/apps/platformer.png`. Also, a copy of it should be in `<AppDir>/platformer.png` and in `<AppDir>/.DirIcon`.

**Comment by 4ian** [comment]
Indeed, we're using a rather old version of electron-builder that might have this fixed for us already. I'll mark this bug as not able to reproduce but still a good first issue if someone wants to try getting this icon to work.

Thanks @probonopd for your investigation :)

**Comment by Silver-Streak** [comment]
@probonopd I'm checking in on older issues. We've had quite a few different updates since then, including some base electron updates.

Are you still experiencing this?

Due to the age of this issue, I'll close this out once I get confirmation of resolution from you, or if no response is received by 09/06/21.
